src/main/java/de/unistuttgart/overworldbackend/data/configuration.java

```diff
@@ -0,0 +1,7 @@
+package de.unistuttgart.overworldbackend.data;
+
+import java.util.List;
+
+public class configuration {
+    public static final List<String> WRITE_ACCESS = List.of("lecturer");
```

Comment:
```suggestion
public final class configuration {
    public static final List<String> WRITE_ACCESS = List.of("lecturer");
    
    private configuration() {}
```

---

Also:
1. what about `Roles` as class name?
2. I don't accept the argument that it isn't obvious that `admin`s have write permission as well, since the calls will always be `<jwt>.hasRolesOrThrow(cookie, Roles.LECTURER)`. It is implied that an admin is also a lecturer.
3. I can even reverse your own argument against you: Assume you want to have an `admin` only route. You could think that anyone with `WRITE_ACCESS` is an admin.